elide-integration-tests/src/test/java/example/Book.java

```diff
@@ -45,6 +47,7 @@
     private String title;
     private String genre;
     private String language;
+    @JsonIgnore
```

Comment:
What's this for?

Reply:
@aklish When I was serialize a book instance Jackson keeps serializing this field so I asks Jackson to ignore it.

Let me put a hold on it and see if I can remove it after the refactoring and tests which I'll finish today